gulp/tasks/scripts.js

```diff
@@ -0,0 +1,19 @@
+var gulp            = require('gulp');
+var config          = require('../config').scripts;
+var browserSync     = require('browser-sync');
+var concat          = require('gulp-concat');
+var rename          = require('gulp-rename');
+var uglify          = require('gulp-uglify');
+
+gulp.task('scripts', ['pl_scripts-lint'], function() {
+    return gulp.src([
+            // setup script sequence
+            './_src/vendor/svg4everybody/svg4everybody.min.js' // polyfill for SVGs in IE9-11
+        ])
+        .pipe(concat('vendor.js'))
```

Comment:
Hmmm, eventually this will mirror the ``pl_scripts.js`` task and we won't be just concatenating vendor files but vendor files and then any custom work/files to one file. WIth that in mind, I'd prefer that we stick with the convention of having the concatenated file be named ``main.js``. In this case this would eventually be minified and placed in ``/public/js/main.min.js``.

Can you sync up that name?